src/main/java/seedu/java/util/TaskList.java

```diff
@@ -0,0 +1,134 @@
+package seedu.java.util;
+
+import java.util.ArrayList;
+
+/**
+ * Handles Task and their operations.
+ *
+ * ArrayList<Task> getTaskArr()
+ * String listToPrint()
+ * String read()
+ */
+public class TaskList {
+    private ArrayList<Task> taskArr;
+    private String path = "data/duke.txt";
+
+    public TaskList() {
+        this.taskArr = new ArrayList<>();
+    }
+
+    public TaskList(ArrayList<Task> taskArr) {
+        this.taskArr = taskArr;
+    }
+
+    /**
+     * Returns its ArrayList of Tasks
+     * @return taskArr
+     */
+    public ArrayList<Task> getTaskArr() {
+        return taskArr;
+    }
+
+    private String totalTask() {
+        return "You have " + (taskArr.size()) + " task on your list.";
+    }
+
+    private String addTodo(String task) {
```

Comment:
Again, possible violation of Open-Closed Principle where a new method will need to be created for every new Task! I feel that it would be better to have a general `addTask(Task t)` method to add all possible Tasks